qiita_pet/templates/sitebase.html

```diff
@@ -239,7 +239,8 @@
       {% block content %}{% end %}
     </div>
 
-    <div id="footer">
+    <div id="footer" style="font-size: 12px; text-align: center; z-index: 1000;">
+      Thank you for using Qiita. For citations point to <a href="http://qiita.microbio.me">http://qiita.microbio.me</a>.
```

Comment:
Interesting, Chrome thinks this page is in Portuguese...

It may be the case that in the future there are publications built on top of portals as a resource paper. But that can be resolved then I guess.

On an aside, any reason qiita is not using `https`?